File: service/php-symfony/src/Entity/ListEntity.php
<?php

namespace App\Entity;

use App\Repository\ListEntityRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Uid\UuidV4;

#[ORM\Entity(repositoryClass: ListEntityRepository::class)]
#[ORM\Table(name: 'list')]
class ListEntity
{
    #[ORM\Id]
    #[ORM\Column(type: 'uuid', unique: true)]
    #[ORM\GeneratedValue(strategy: "CUSTOM")]
    #[ORM\CustomIdGenerator(class: 'doctrine.uuid_generator')]
    private $id;

    #[ORM\ManyToOne(targetEntity: Account::class, inversedBy: 'lists')]
    private $Account;

    #[ORM\Column(type: 'text', nullable: true)]
    private $name;

    #[ORM\Column(type: 'datetimetz', nullable: true)]
    private $creation_date;

    public function getId(): ?UuidV4
    {
        return $this->id;
    }

    public function getAccount(): ?Account
    {
        return $this->Account;
    }

    public function setAccount(?Account $Account): self
    {
        $this->Account = $Account;

        return $this;
    }

    public function getName(): ?string
    {
        return $this->name;
    }

    public function setName(?string $name): self
    {
        $this->name = $name;

        return $this;
    }

    public function getCreationDate(): ?\DateTimeInterface
    {
        return $this->creation_date;
    }

    public function setCreationDate(?\DateTimeInterface $creation_date): self
    {
        $this->creation_date = $creation_date;

        return $this;
    }
}

File: service/jvm-java-quarkus/src/main/java/com/ecobenchmark/entities/ListEntity.java
package com.ecobenchmark.entities;


import com.ecobenchmark.controllers.getlists.ListResponse;
import com.ecobenchmark.controllers.getlists.TaskResponse;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;
import java.time.Instant;
import java.util.List;
import java.util.UUID;

@Entity(name = "list")
@SqlResultSetMapping(
        name = "getListsResponseMapping",
        classes = {
                @ConstructorResult(
                        targetClass = ListResponse.class,
                        columns = {
                                @ColumnResult(name = "id" ,type = UUID.class),
                                @ColumnResult(name = "name", type = String.class),
                                @ColumnResult(name = "creation_date", type = Instant.class),
                                @ColumnResult(name = "account_id", type = UUID.class)}),
                @ConstructorResult(
                        targetClass = TaskResponse.class,
                        columns = {
                                @ColumnResult(name = "task_id" ,type = UUID.class),
                                @ColumnResult(name = "task_name", type = String.class),
                                @ColumnResult(name = "task_description", type = String.class),
                                @ColumnResult(name = "task_creation_date", type = Instant.class)})
        })
public class ListEntity {

    @Id
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    private UUID id;

    @ManyToOne
    private Account account;

    private String name;

    @Column(name = "creation_date")
    private Instant creationDate;

    @OneToMany(mappedBy = "list")
    private List<Task> tasks;

    public ListEntity(Account account, String name, Instant creationDate, List<Task> tasks) {
        this.account = account;
        this.name = name;
        this.creationDate = creationDate;
        this.tasks = tasks;
    }

    public ListEntity() {
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public Account getAccount() {
        return account;
    }

    public void setAccount(Account account) {
        this.account = account;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Instant getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Instant creationDate) {
        this.creationDate = creationDate;
    }

    public List<Task> getTasks() {
        return tasks;
    }

    public void setTasks(List<Task> tasks) {
        this.tasks = tasks;
    }
}
